fispact_output_reader.py

```diff
@@ -241,10 +241,18 @@ def read_summary_data(data):
     inhal = []
     inhal_un = []
     trit = []
+    cooling = []
     to = 0
+    is_cooling = True
 
     for line in sum_lines:
         if isFisII(data):
+            if "Irradn" in line:

```

Comment:
This should be finished now, thank you for clarifying. Pandas really didn't like me adding only selected time steps to the data frame (since all columns need to be the same length), so I've stored any timesteps from the irradiation phase as Nan in the cooling phase timesteps column. Hopefully this shouldn't affect the plotting in any way. I have also created a test to see that it was putting Nans and actual numbers in the right indices within the column and it all worked as expected :)

Reply:
definately need to read all the time steps, the selection to only cooling should be done later. best practice for this sort of file reading is to read everything into appropriate data structures then apply filtering later as needed.

Reply:
![image](https://github.com/user-attachments/assets/41074237-c225-4245-8ed3-83b0b2767e32)
Sorry, I think I need a bit more clarification on how best to approach this. Should I instead create more data frames with only data from the cooling/irradiation phase in them respectively? That way you read and retain the original data frame, with the option to plot one phase at a time. Also, there is a data point within the irradiation phase that is classified as cooling (image attached), and I'm not sure how to classify this if I was to create separate data frames.